pkgs/servers/pulseaudio/default.nix

```
@@ -143,6 +143,9 @@ stdenv.mkDerivation rec {
     + ''
     moveToOutput lib/cmake "$dev"
     rm -f $out/.bin-unwrapped/qpaeq # this is packaged by the "qpaeq" package now, because of missing deps
+
+    mkdir -p $config
+    cp config.h $config
```

```suggestion
    moveToOutput config.h $config
```
will that work? Also shouldn't that just be in the dev output?

---

Dropped